src/lotus/client.rs

```diff
@@ -212,4 +221,35 @@ impl<T: JsonRpcClient + Send + Sync> LotusClient for LotusJsonRPCClient<T> {
         log::debug!("received chain_head response: {r:?}");
         Ok(r)
     }
+
+    async fn ipc_get_prev_checkpoint_for_child(
+        &self,
+        child_subnet_id: SubnetID,
+    ) -> Result<IPCGetPrevCheckpointForChildResponse> {
+        let parent = match child_subnet_id.parent() {
+            None => return Err(anyhow!("The child_subnet_id must be a valid child subnet")),
+            Some(parent) => parent,
+        };
+        let subnet_actor = child_subnet_id.subnet_actor().to_string();
+        let params =
+            json!([GATEWAY_ACTOR_ADDRESS, {"Parent": parent.to_string(), "Actor": subnet_actor}]);
+        println!("{}", params);
```

Comment:
probably a missed `println` for log::info?

Reply:
oops